src/main/resources/staff/templates/email/claimantResponse/rejection/directionsHearing/body.txt

```diff
@@ -0,0 +1,3 @@
+The claimant has reviewed the defendant’s response and wish to proceed with the claim if they opted into mediation the MILO team has been sent their contact & claim details.
```

Comment:
Was this content reviewed? Could use some punctuation and grammatical changes

Reply:
yes reviewed by PO.